Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -140,14 +140,16 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
          * @param {EventFacade} e
          */
         _selectLanguage: function (e) {
+            var isNewContent = this.get('content').isNew();
```

Comment:
I'm not sure what is the relation between `isNewContent` and the `translationMode`. Maybe you should rename variable to make it more meaningful.